packages/@sanity/types/src/schema/definition/type/array.ts

```diff
@@ -6,7 +6,23 @@ import {
   type IntrinsicTypeName,
   type TypeAliasDefinition,
 } from '../schemaDefinition'
-import {type BaseSchemaDefinition, type SearchConfiguration, type TitledListValue} from './common'
+import {
+  type BaseSchemaDefinition,
+  type FieldGroupDefinition,
+  type SearchConfiguration,
+  type TitledListValue,
+} from './common'
+
+/** @public */
```

Comment:
Would also mark this as alpha, so it's clear that if anyone imports `import type {InsertMenuOptions} from '@sanity/types'` then they are prepared that it might change in breaking ways even in minor or patch releases 🙌 

```suggestion
/** @alpha This API may change */
```

Reply:
Thank you, I've now marked this interface as `@alpha` as well.